docs/Reference.md

```diff
@@ -831,9 +831,9 @@ If true, this will be able horizontal swipe gestures when using the WKWebView. T
 
 Does not store any data within the lifetime of the WebView.
 
-| Type    | Required | Platform      |
-| ------- | -------- | ------------- |
-| boolean | No       | iOS WKWebView |
```

Comment:
Maybe keep the Platform section and update it to `iOS WKWebView, Android` so users know that this boolean only works with a WKWebView and not a normal one on iOS.